ops/scripts/release-bus-v2-fast-off.mjs

```
@@ -0,0 +1,270 @@
+
+import { spawnSync } from 'node:child_process';
+
+const EXPECTED_ACCOUNT = '987989283142';
+const REPOSITORIES = [
+  '6529-Collections/6529seize-backend',
+  '6529-Collections/6529seize-frontend'
+];
+const FUNCTIONS = ['seizeAPI', 'releaseBusV2Reconciler'];
+const RULE_PREFIX = 'releaseBus-prod-V2ReconcilerEventsRuleSchedule1';
+const API_URL =
+  process.env.RELEASE_BUS_API_URL?.trim() || 'https://api.6529.io';
```

ESLint

[error] 'process' is not defined.

(no-undef)